lib/apiservers/engine/backends/container.go

```diff
@@ -1156,12 +1119,12 @@ func interBridgeTraffic(op portmap.Operation, hostPort, proto, containerAddr, co
 		// chain rather than appended to supersede bridge-to-bridge
 		// traffic blocking
 		baseArgs := []string{"-t", string(iptables.Filter),
-			"-i", bridgeIfaceName,
-			"-o", bridgeIfaceName,
-			"-p", proto,
-			"-d", containerAddr,
-			"--dport", containerPort,
-			"-j", "ACCEPT",
+				     "-i", bridgeIfaceName,
```

Comment:
I recommend just reverting this change since it is a blank space change set. will make the pr a little easier to read.